src/utils/selectors.ts

```diff
@@ -58,19 +59,33 @@ export const currentVenueSelectorData: SparkleSelector<AnyVenue | undefined> = (
  *
  * @param state the Redux store
  */
-export const worldUsersSelector: SparkleSelector<WithId<User>[] | undefined> = (
-  state
-) => state.firestore.ordered.worldUsers;
+export const worldUsersSelector: SparkleSelector<
+  WithId<UserWithLocation>[] | undefined
+> = (state) => state.firestore.ordered.worldUsers;
+
+export const worldUsersWithoutLocationSelector: SparkleSelector<
+  WithId<User>[] | undefined
+> = (state) => worldUsersSelector(state)?.map(userWithLocationToUser);
```

Comment:
the name `userWithLocationToUser` implies that user will have location?

Reply:
See my explanation to @denisdimitrov about this in https://github.com/sparkletown/sparkle/pull/1703#discussion_r664153565

it takes a `userWithLocation` and makes it a `user`.

Reply:
> See my explanation to @denisdimitrov about this in #1703 (comment)
> 
> it takes a userWithLocation and makes it a user.

Given that it brought attention of both me and @denisdimitrov, it can be a good sign that the naming is suboptimal

Reply:
It's a fairly common pattern to use for 'converter' type functions. eg.

https://date-fns.org/v2.22.1/docs/hoursToSeconds

[OOS/NB] I'm open to suggestions for better naming.